{"name": "test_32应急取款成功", "status": "broken", "statusDetails": {"message": "uiautomator2.exceptions.XPathElementNotFoundError: //*[@text=\"您已经委托成功\"]", "trace": "self = <建投自动化测试.现金管理测试用例.test_现金管理-智多鑫-应急取款.Test_现金管理_智多鑫_应急取款 object at 0x11a856f10>\nu2driver = uiautomator2 object for 127.0.0.1:63870\n\n    @pytest.mark.cashmanage\n    @allure.story(\"应急取款成功\")\n    def test_32应急取款成功(self, u2driver):\n        \"\"\"应急取款成功\"\"\"\n        cp = cashproductpage(u2driver)\n        cp.emergencyamount_zdx1_sendkey()\n        cp.emergencywithdrawal_click()\n        cp.definebtn_click()\n        time.sleep(2)\n>       at = cp.successtip_text()\n\ntest_现金管理-智多鑫-应急取款.py:143: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../PageObject/cashproductpage.py:336: in successtip_text\n    return self.text(self.successtip)\n../../Common/PageBaseU2.py:143: in text\n    with allure.step(\"获取{0}文本信息：{1}\".format(element[2],el.get_text())):\n/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/uiautomator2/xpath.py:440: in get_text\n    return self.get().attrib.get(\"text\", \"\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <uiautomator2.xpath.XPathSelector object at 0x11a7d7510>\n\n    def get(self):\n        \"\"\"\n        Get first matched element\n    \n        Returns:\n            XMLElement\n    \n        Raises:\n            XPathElementNotFoundError\n        \"\"\"\n        if not self.wait(self._global_timeout):\n>           raise XPathElementNotFoundError(self._xpath)\nE           uiautomator2.exceptions.XPathElementNotFoundError: //*[@text=\"您已经委托成功\"]\n\n/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/uiautomator2/xpath.py:424: XPathElementNotFoundError"}, "description": "应急取款成功", "steps": [{"name": "在取款金额输入框中输入1000", "status": "passed", "attachments": [{"source": "42c82517-a421-4b5f-8e7a-eaf68da1fb38-attachment.png", "type": "image/png"}], "start": 1571910482951, "stop": 1571910482952}, {"name": "点击确定", "status": "passed", "attachments": [{"source": "c1e21a13-c5de-443b-bdca-73c6f8af8756-attachment.png", "type": "image/png"}], "start": 1571910486006, "stop": 1571910486007}], "attachments": [{"name": "log", "source": "ff818e97-39aa-4c0d-b76b-bb07926dfa96-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "0382e7e6-e230-4ace-a020-7e1162d39445-attachment.txt", "type": "text/plain"}], "start": 1571910481967, "stop": 1571910518377, "uuid": "ef7151db-efb0-4ff8-aff9-4a7165b2a933", "historyId": "7711bf3e09c55a30a2c9139811eda5e5", "testCaseId": "b6c30c8b9442511f74cd0a4fe4a729ea", "fullName": "建投自动化测试.现金管理测试用例.test_现金管理-智多鑫-应急取款.Test_现金管理_智多鑫_应急取款#test_32应急取款成功", "labels": [{"name": "story", "value": "应急取款成功"}, {"name": "feature", "value": "现金管理-智多鑫-应急取款用例"}, {"name": "tag", "value": "cashmanage"}, {"name": "parentSuite", "value": "建投自动化测试.现金管理测试用例"}, {"name": "suite", "value": "test_现金管理-智多鑫-应急取款"}, {"name": "subSuite", "value": "Test_现金管理_智多鑫_应急取款"}, {"name": "host", "value": "localhost"}, {"name": "thread", "value": "8061-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "建投自动化测试.现金管理测试用例.test_现金管理-智多鑫-应急取款"}]}